tests/test_client.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import rdflib
+import requests
+
+from fdpclient.client import Client
+
+# TODO
+# - add tests on fdp
+
+# NOTE: change IP address to domain name after solving FDP server config issue
+base_url = 'http://145.100.57.30'
+catalogID = 'catalog01'
+data_url = base_url + '/catalog/' + catalogID
+
+@pytest.fixture(scope='class', autouse=True)
+def client():
+    r = requests.delete(data_url)
```

Comment:
Wait, so you're connecting to a public server and then deleting data?

Reply:
Hmmm, it's a test/demo server :-)

Reply:
Ah, that helps :). What if two people run the tests simultaneously though? Or, more likely, that you set up a couple different CI scenarios that end up running in parallel?